src/main/java/org/cbio/gdcpipeline/tasklet/SetUpPipelineTasklet.java

```diff
@@ -42,6 +42,7 @@ public RepeatStatus execute(StepContribution stepContribution, ChunkContext chun
 
     private void initiateCancerStudy() {
         supportedCancerStudy.add("TCGA_BRCA");
```

Comment:
let's move the supported studies to a property instead or add a TO-DO for later. spring can interpret string arrays as properties
```
gdc.supported_studies=TCGA_BRCA,TCGA_PCPG
```

Reply:
I think it should totally be deleted since we're using the api now. The only reason this existed was because of the xml schema differences (I think)